### PR TITLE
most_accurate_team/least_accurate_team

### PR DESCRIPTION
**Pull request by zach-bergman**
## Type Of Change
- [X] New Feature
- [ ] Debugging
- [ ] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change
Added most_accurate_team and least_accurate_team methods, along with helpers. Tests for all, followed TDD.

## Group Project Check List
- [X] I made test for my code, and made sure they were passing before sucmitting PR
- [X] I assigned atleast 2 group members to review my PR

### CODE REVIEW

File: lib/game_team.rb [hunk 1]
```diff
@@ -22,8 +22,8 @@ def initialize(gameteam_data)
         @result = gameteam_data[:result]
         @settled_in = gameteam_data[:settled_in]
         @head_coach = gameteam_data[:head_coach]
-        @goals = gameteam_data[:goals]
-        @shots = gameteam_data[:shots]
+        @goals = gameteam_data[:goals].to_i
```
Comment (austincarrjones) [inline lib/game_team.rb:25]: I did this as well in my PR

**Review by austincarrjones — APPROVED** [review]
beastly methods! TBH I haven't fully comprehended them, might be cool to review over a call.

**Review by noahdurbin — APPROVED** [review]
looks good.

File: spec/game_team_spec.rb [hunk 2]
```diff
@@ -68,7 +68,7 @@
       end
     end
 
-    describe "#coach_win_count - helper" do
+    describe "::coach_win_count - helper" do
```
Comment (noahdurbin) [inline spec/game_team_spec.rb:71]: nice to know that there is a notation to specify what tests are here for.

**Comment by noahdurbin** [comment]
Had to solve a couple merge conflicts on this one, I think I was able to do it all right but we will want to double check that everything runs as expected when we pull from main.

**Pull request merged**